docs/NLUGitHubActions.md

```diff
@@ -0,0 +1,85 @@
+2. send a test set to created LUIS model
+3. evaluate model by comparing results received from LUIS with expected values
+4. delete the model from the portal
+
+Let's name this pipeline CINLU and run it on a pull request to GitHub repo.
+
+```
+name: CINLU
+
+on: [pull_request]
+```
+
+1. Let's install NLU.DevOps cli tool on GitHub agent.
+
+```
+    - name: Install dotnet-nlu
+      run: dotnet tool install -g dotnet-nlu
+```
+On the ubuntu agent, you need to prepend a directory to the system PATH variable for all subsequent actions in the current job to make sure that cli tool works. More about this command [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#add-a-system-path-add-path).
+
+```
+    - name: Path
+      run: echo "::add-path::$HOME/.dotnet/tools"
+```
+We use [utterances.json](../models/utterances.json) for training. You can replace this file with another file that consists of intents, utterances, entities that you need for your own model.
+More about the format of this file [here](https://github.com/microsoft/NLU.DevOps/blob/master/docs/GenericUtterances.md).
+To train your model we should add the following:
+
+```
+- name: Train Luis model
+  run: dotnet nlu train -s luis -u utterances.json --save-appsettings
+  env:
+    luisAuthoringRegion: ${{ secrets.luisAuthoringRegion }}
+    luisAuthoringKey: ${{ secrets.luisAuthoringKey }}
+ ```
+
+More about the command [here](https://github.com/microsoft/NLU.DevOps/blob/master/docs/Train.md). 
+Before you push to the repo, you need to add credentials (at least luisAutoringKey and luisAuthoringRegion are required) to your GitHub Secrets. For example,
+![credentials](./images/credentials.png)
+
+You can open the Azure portal and copy LUIS Authoring Cognitive Service region and key. More about LUIS authoring resources [here](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-how-to-azure-subscription).
+![luisAuthAzure](./images/luisAuthAzure.png)
+
+You can find this information on LUIS.ai portal as well.
+![luisCredentials](./images/luiscred.png)
+
+where `Region` -> `luisAuthoringRegion` and `Primary key` -> `luisAuthoringKey`
+
+2. To test LUIS model let's use [utterancesTest.json](../models/utterancesTest.json) file.
+We can save the result in results.json file using `--save-appsettings` for training. We don't need then to provide additional information to the `test` command. 
+Yaml may look like that:
+```
+    - name: Test Luis model
+      run: dotnet nlu test -s luis -u utterancesTest.json -o results.json
+      env: 
+        luisAuthoringRegion: ${{ secrets.luisAuthoringRegion }}
+        luisAuthoringKey: ${{ secrets.luisAuthoringKey }}
+```
+3. To evaluate results we use two files: `utterancesTest.json` that consists of utterances that are labeled with actual intents and entities and `results.json` file with intent and entity names/types that we get from created LUIS model.
```

Comment:
I would focus on the action we're taking rather than the files we're using. Something like:
"We use the `compare` command from the NLU.DevOps CLI to evaluate the results. The expected intents and entities in this case are given in the `utterancesTest.json` file, while the results predicted by the LUIS model are in the `results.json` file."